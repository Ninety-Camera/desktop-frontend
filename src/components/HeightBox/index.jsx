import React from "react";

export default function HeightBox(props) {
  const { height } = props;
  return <div style={{ height: height }} />;
}
