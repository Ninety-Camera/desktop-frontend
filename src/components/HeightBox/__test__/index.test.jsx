import React from "react";
import { render, screen, cleanup } from "@testing-library/react";
import ReactDom from "react-dom";
import HeightBox from "..";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import { useState } from "react";

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDom.render(<HeightBox />, div);
});

it("renders form correctly", () => {
  //use for props passing
  const { getByTestId } = render(<HeightBox height={20} />);
  //expect(getByTestId("blackHorizontalBar")).toHaveTextContent("Ninety Camera");
});

it("matches snapshot", () => {
  const tree = renderer.create(<HeightBox height={20} />).toJSON();
  expect(tree).toMatchSnapshot();
});
