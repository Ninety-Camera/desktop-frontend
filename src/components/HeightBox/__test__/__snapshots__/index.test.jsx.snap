// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  data-testid="heightBox"
  style={
    Object {
      "height": 20,
    }
  }
/>
`;
